notebooks/02-ingestion/05-partitioning-data.ipynb

```diff
@@ -0,0 +1,717 @@
+    "print (f\"{use_table} {test_in_parallel(sql.format(use_table), 4, 50)}\")\n",
+    "\n",
+    "use_table=\"flights_year_IATA\"\n",
+    "print (f\"{use_table} {test_in_parallel(sql.format(use_table), 4, 50)}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddaf1e58-ba7a-49e2-8cda-60fc52979d6f",
+   "metadata": {},
+   "source": [
+    "My results:\n",
+    "```\n",
+    "flights_year_50k Results  = count:200  avg:14.543 ms   min:11.341 ms  max:68.145 ms\n",
+    "flights_year_IATA Results = count:200  avg:13.909 ms   min:11.019 ms  max:33.341 ms\n",
+    "```\n",
+    "The avg continues to grow, meaning that we've likely hit some limit in resources. The minimum continues unchanged which also makes sense because the first time the query runs it probably doesn't need to wait. Subsequent parallel requests will need to wait in a queue while resources are released, as we increate the concurrency, there's more queries in the queue.\n",
+    "\n",
+    "The important difference continues to be in the max value that is now `2.06` x faster on the IATA table.\n",
+    "Let's get one more data point...\n",
+    "\n",
+    "#### Concurrency = 8"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ec00d899-7f25-4254-ab23-b79a7920e9e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "use_table=\"flights_year_50k\"\n",
+    "print (f\"{use_table} {test_in_parallel(sql.format(use_table), 8, 50)}\")\n",
+    "\n",
+    "use_table=\"flights_year_IATA\"\n",
+    "print (f\"{use_table} {test_in_parallel(sql.format(use_table), 8, 50)}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f36ff06e-e2ab-4a7a-8c08-c9966b336347",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "My results:\n",
+    "```\n",
+    "flights_year_50k Results  = count:400  avg:25.969 ms   min:11.595 ms  max:97.475 ms\n",
+    "flights_year_IATA Results = count:400  avg:24.171 ms   min:11.979 ms  max:43.507 ms\n",
+    "```\n",
+    "Again, the avg continues to grow and the minimum continues unchanged.\n",
+    "\n",
+    "The important difference continues to be in the max value that is now `2.25` x faster on the IATA table.\n",
+    "\n",
+    "As we increase concurrency the IATA table continues to be more effective at dealing with concurrency. This effect will be more noticeable in a clustered deployment with more independent resources and as segments sizes grow to their more normal 3-10 million rows each."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44738d6d-cec2-40ad-aaba-998c758c63f4",
+   "metadata": {},
+   "source": [
+    "## Clean up\n",
+    "\n",
+    "Run the following cell to remove the tables created in the database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8082b545-ba7f-4ede-bb6e-2a6dd62ba0d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "druid.datasources.drop(\"flights_hour\")\n",
```

Comment:
Just noting that these table names need to change.

Reply:
changed them all to use "-" instead of "_"